x-pack/legacy/plugins/reporting/export_types/csv/server/__tests__/execute_job.js

```diff
@@ -11,7 +11,7 @@ import nodeCrypto from '@elastic/node-crypto';
 
 import { CancellationToken } from '../../../../common/cancellation_token';
 import { FieldFormatsService } from  '../../../../../../../../src/legacy/ui/field_formats/mixin/field_formats_service';
-import { createStringFormat } from  '../../../../../../../../src/legacy/core_plugins/kibana/common/field_formats/types/string';
+import { StringFormat } from '../../../../../../../../src/plugins/data/common/field_formats/converters/string';
```

Comment:
Please don't forget to fix link here 

src/plugins/data/common/field_formats/converters/string'; - > src/plugins/data/common

Reply:
Done everywhere